docs/json/radarr/cf/truehd-atmos.json

```diff
@@ -14,7 +14,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "True[ .-]?HD"
+        "value": "True[ .-]?HD|W4NK3R|HQMUX"
```

Comment:
wouldn't `-DON` be needed here?

Reply:
Again, restored this line to it's previous state. This CF needs both `TrueHD` and `ATMOS` conditions to match. So, if a release name has both `TrueHD` and `-DON` in the title then the CF will match, which is what we want to achieve.